ic-os/dev-tools/bare_metal_deployment/tools.bzl

```diff
@@ -20,14 +20,22 @@ def launch_bare_metal(name, image_zst_file):
             requirement("simple-parsing"),
             requirement("tqdm"),
         ],
-        data = [image_zst_file, "//rs/ic_os/setupos-inject-configuration", "//ic-os/dev-tools/bare_metal_deployment:find_idrac_package_path"],
+        tags = ["manual"],
+    )
+
+    native.sh_binary(
+        name = name,
+        srcs = ["//toolchains/sysimage:proc_wrapper.sh"],
         args = [
+            "python3",
+            "$(location :" + name + "_main)",
```

Comment:
nit: Will bazel let us use consistent formatting for both `"$(location :" + name + "_main)",` and `":%s_main" % name`?

Reply:
Bazel doesn't care but it's a good point, I'll use a variable for consistency.